.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,7 +11,7 @@ body:
       - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md)
       - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/FAQ.md)
       - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
-      - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-core/docs/FAQ.md)
+      - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/lib/msal-core/docs/FAQ.md)
```

Comment:
Question: If we're referring to browser as `MSAL.js` below, and core as `MSAL.js v1`, would we not want to specify this as msal v1 as well here?

Reply:
I don't have a preference. I think this is more just matching the package name.